### PR TITLE
Change base language of html-files to Norwegian

### PR DESCRIPTION
**Pull request by henrikhorluck**
## What kind of a pull request is this?

- QA / Code Review

## Code Checklist

- [X] The code follows dotkom code style 
- [X] The code passes the defined tests
- [ ] I have added tests for the code I added
- [ ] I have provided documentation for the code I added
- [X] The code is ready to be merged

## (Possible) Breaking Changes

- [X] The changes are backwards compatible

- [ ] I have updated the build configuration
- [ ] These changes requires changes to configuration in production 
    - [ ] I have applied the required changes in production
    - [ ] I cannot apply the required changes in production before this is deployed.

## Description of changes
Made the lang-tag of base html-pages to "nb"/Norwegian Bokmaal, so that for example Chrome doesn't ask if you want to translate the page from Danish or something similar.

### CODE REVIEW

**Comment by sklirg** [comment]
What about `nb-NO`? both bokmål and norwegian.

https://www.w3schools.com/tags/ref_language_codes.asp

**Comment by FredrikAugust** [comment]
As far as I can tell, that is not a valid ISO 639-2 code, which is the specification HTML uses for the language tag.

Norwegian (`no`) is considered a macro language[0], or a superset of Bokmål and Nynorsk if you want, and thus it encompasses both cases. `nb-NO` would if it existed (at least I think it doesn't exist in this in this spec.) be reduntant, as `nb` is already inferred by `no`. It's like answering "Yes. I like cats and felines".

Your suggestion looks like a `locale` from the linux-realm (a combo of ISO 3966 and ISO 639) (e.g. `nb_NO`).

Finally I want to add that it's not impossible that someone comes along and posts something in Nynorsk, which would break your purist scheme (\s). Remember that we have two official written languages, and as a student organization, we should support them both.

**Comment by henrikhorluck** [comment]
The `nb-NO` is a part of best current practice (BCP 47) [0], also with a quick Google search you will see that there is absolutely zero Nynorsk articles on OW today [1], and I doubt there will be more, according to the development of Nynorsk usage today [2], but of course if someone wants to write an article in Nynorsk, it it wouldn't be world ending to have one article with the wrong language tag. As it is today, `nb-NO` is the most accurate tag for our current content.

[[0] https://www.ietf.org/rfc/bcp/bcp47.txt](https://www.ietf.org/rfc/bcp/bcp47.txt)
[[1] https://www.google.no/search?ei=EojZW4vQEMahsAGzi4ygDg&q=site%3Awww.online.ntnu.no+ein+OR+eg+OR+ikkje+OR+ei+OR+eit&oq=site%3Awww.online.ntnu.no+ein+OR+eg+OR+ikkje+OR+ei+OR+eit&gs_l=psy-ab.3...6593.55644.0.55838.19.10.9.0.0.0.120.913.9j1.10.0....0...1c.1.64.psy-ab..0.0.0....0.54tiWinxQL0](https://www.google.no/search?ei=EojZW4vQEMahsAGzi4ygDg&q=site%3Awww.online.ntnu.no+ein+OR+eg+OR+ikkje+OR+ei+OR+eit&oq=site%3Awww.online.ntnu.no+ein+OR+eg+OR+ikkje+OR+ei+OR+eit&gs_l=psy-ab.3...6593.55644.0.55838.19.10.9.0.0.0.120.913.9j1.10.0....0...1c.1.64.psy-ab..0.0.0....0.54tiWinxQL0)
[[2] http://www.ssb.no/statbank/sq/10013467/](http://www.ssb.no/statbank/sq/10013467/)

**Comment by duvholt** [comment]
Just a note about nynorsk articles on OW:

If you scroll down to 2014-2015 in the article archive you'll find a bunch of articles written in nynorsk. :smile:  
https://online.ntnu.no/article/

**Comment by FredrikAugust** [comment]
I just re-googled it, and it turns out you are correct. The reason I was confused was because I used the _satanic hell spawn_ called w3schools, because for some reason, even though I spend 99% of my life force preventing it, it ends up on top of my google results.

However, upon checking my absolute favorite website on the planet, Microsoft Developer Network, it turns out that ISO 639-2 is **not** the standard, even though ~~satan~~ w3schools claims it. It is in fact, as you wrote, BCP 47.

tl;dr fuck w3schools

**Comment by FredrikAugust** [comment]
So yes, keep `nb-NO`.

**Comment by FredrikAugust** [comment]
And as there is no downside to merging this, and Lord Sklirg has approved of it, I will merge it. Thank you for your contribution.

**Comment by henrikhorluck** [comment]
Is there a way for article-writers on OW to write direct HTML, so that they could add their own language tag?

**Comment by FredrikAugust** [comment]
I don't believe so, though it is possible to slap the attribute on any tag, so we could implement a simple API for setting the language of an article.

**Comment by sklirg** [comment]
Språket i HTML sin lang-attributt skal være hovedspråket for siden. Eventuelle avvikninger fra dette kan spesifiseres med ytterligere lang-attributter i teksten. Se her for info: https://uu.difi.no/krav-og-regelverk/losningsforslag-web/sprak-i-koden
